app/controllers/projects_controller.rb

```diff
@@ -2,17 +2,19 @@ class ProjectsController < ApplicationController
   before_action :authenticate_user!
   before_action :set_project, only: %i[show edit destroy]
   before_action :check_contributor, only: %i[show edit destroy]
+  before_action :set_project_job_categories, only: %i[show]
```

Comment:
AChei um pouco desnecessário por só estar em uma `action`, mas nada impeditivo pra seguirmos assim.

Reply:
Acredito que não percebemos pelo fato de estarmos acostumados a setar o `id` de alguma tabela sempre em `before_action` kkk. Se a gente não for usar em outra `action` futuramente, podemos retirar  e implementar diretamente no método.